domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/abakus/ArbeidsforholdTjeneste.java

```diff
@@ -81,7 +81,7 @@ private static ArbeidsforholdMedPermisjon mapArbeidsforholdMedPermisjon(Arbeidsf
         return new ArbeidsforholdMedPermisjon(
             mapTilArbeidsgiver(dto),
             no.nav.foreldrepenger.behandlingslager.virksomhet.ArbeidType.fraKode(dto.getType().getKode()),
-            EksternArbeidsforholdRef.ref(dto.getArbeidsforholdId().getEksternReferanse()),
+            EksternArbeidsforholdRef.ref(dto.getArbeidsforholdId() != null ? dto.getArbeidsforholdId().getEksternReferanse() : null ),
```

Comment:
Foreslår 
`dto.getArbeidsforholdId() != null ? EksternArbeidsforholdRef.ref(dto.getArbeidsforholdId().getEksternReferanse()) : EksternArbeidsforholdRef.nullRef()`

Reply:
Er det ikke like greit med en null siden ArbeidsforholdMedPermisjon forventer en String?

Reply:
Det er identisk. Ved å bruke EksternArbeidsforholdRef.nullRef() så bruker du en metode som brukes flere steder. Gjør det enklere med sporbarhet på hva som er null og ikke. Men det er bare småpirk. OK for meg :)

Reply:
Endret til å bruke nullRef.getReferanseI()

Reply:
Tenkte mer på formatet 
dto.getArbeidsforholdId() != null ? 
EksternArbeidsforholdRef.ref(dto.getArbeidsforholdId().getEksternReferanse())
EksternArbeidsforholdRef.nullRef()
👼